src/resmom/mom_main.c

```diff
@@ -504,6 +506,7 @@ static struct	specials {
 	{ "alien_kill",			set_alien_kill },
 #if	MOM_ALPS
 	{ "alps_client",		set_alps_client },
+	{ "vnode_per_numa_node", 	set_vnode_per_numa_node },
```

Comment:
Nit: If you have to touch the file again, you could make the order here match the order at lines 445-448.